rosdep/python.yaml

```diff
@@ -200,6 +200,14 @@ ipython3:
   debian: [ipython3]
   fedora: [ipython3]
   ubuntu: [ipython3]
+jupyter-nbconvert:
+  arch: [jupyter-nbconvert]
+  debian: [jupyter-nbconvert]
+  fedora: [jupyter-nbconvert]
```

Comment:
From the link I posted earlier, the Fedora key should be `python3-nbconvert`

Reply:
Thank you!